examples/model_evaluation/plot_feature_importance.py

```diff
@@ -0,0 +1,551 @@
+# - The root mean squared error (:func:`~sklearn.metrics.root_mean_squared_error`),
+#   abbreviated as RMSE, which is an error.
+#   It takes the square root of the mean squared error (MSE) so it is expressed in the
+#   same units as the target variable.
+#   The MSE measures the average squared difference between the predicted values and
+#   the actual values.
+#
+# Here, the :math:`R^2` seems quite poor, so some further preprocessing would be needed.
+# This is done further down in this example.
+# For now, keep in mind that any observations drawn from inspecting the coefficients
+# of this simple Ridge model are made on a model that performs quite poorly, hence
+# must be treated with caution.
+
+# %%
+# To inspect our model, let us use the
+# :meth:`skore.EstimatorReport.feature_importance` accessor:
+
+# %%
+ridge_report.feature_importance.coefficients()
+
+# %%
+# .. note::
+#   More generally, :meth:`skore.EstimatorReport.feature_importance` can help you
+#   inspect the coefficients of all linear models.
+#   We consider a linear model as defined in
+#   `scikit-learn's user guide
+#   <https://scikit-learn.org/stable/modules/linear_model.html>`_.
+#   In short, we consider a "linear model" as a scikit-learn compatible estimator that
+#   holds a ``coef_`` attribute (after being fitted).
+
+# %%
+# Since we have included scaling in the pipeline, the resulting coefficients are all on
+# the same scale, making them directly comparable to each other.
+# Without this scaling step, the coefficients in a linear model would be influenced by
+# the original scale of the feature values, which would prevent meaningful comparisons
+# between them.
+#
+# .. seealso::
+#
+#   For more information about the importance of scaling,
+#   see scikit-learn's example on
+#   `Common pitfalls in the interpretation of coefficients of linear models
+#   <https://scikit-learn.org/stable/auto_examples/inspection/plot_linear_model_coefficient_interpretation.html>`_.
+#
+# Here, it appears that the ``MedInc``, ``Latitude``, and ``Longitude`` features are
+# the most important, with regards to the absolute value of other coefficients.
+# This finding is consistent with our previous observations from the *Associations*
+# tab of the table report.
+#
+# However, due to the scaling, we can not interpret the coefficient values
+# with regards to the original unit of the feature.
+# Let us unscale the coefficients, without forgetting the intercept, so that the
+# coefficients can be interpreted using the original units:
+
+# %%
+import numpy as np
+
+mu = ridge_report.estimator_[0].mean_
+sigma = ridge_report.estimator_[0].scale_
+
+
+def unscale_coefficients(df, mu, sigma):
+    df.loc["Intercept", "Coefficient"] = df.loc["Intercept", "Coefficient"] - np.sum(
+        df.loc[df.index != "Intercept", "Coefficient"] * mu / sigma
+    )
+    df.loc[df.index != "Intercept", "Coefficient"] = (
```

Comment:
It would be easier to read if you store in a variable the column that you try to indexed, e.g.:

```python
mask_intercept_column = df.index == "intercept"
```